### PR TITLE
chore: update docker files to 0.3.0

### PR DESCRIPTION
**Pull request by douglaz**

### CODE REVIEW

**Comment by Kodylow** [comment]
tACK both guardian (solo + 2 federations) and gateway

**Review by Kodylow — APPROVED** [review]
tACK

File: docker/0.3.0/full-tls-mutinynet/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+        VirtualHost "xmpp.fedimint.my-super-host.com"
+          enabled = true;
+        EOF
+        tail -f /dev/null
+    volumes:
+      - prosody_datadir:/prosody_data
+    restart: always
+
+  xmpp:
```
Comment (elsirion) [inline docker/0.3.0/full-tls-mutinynet/docker-compose.yaml:334]: I think that shouldn't be in here

**Review by elsirion — APPROVED** [review]
I think it would be nice to have a version that is just a guardian server with TLS proxy and admin UI on the same domain. That's probably the most common use case.

How we can achieve single-domain:
* `mydomain.com`: admin UI
* `mydomain.com/ws`: websocket API of fedimintd

I've used this with a Nix-based setup in the past and it works well.

**Review by dpc — APPROVED** [review]

**Pull request merged**